tools/src/main/scala/scala/scalanative/linker/Reach.scala

```diff
@@ -544,8 +544,9 @@ class Reach(config: build.Config, entries: Seq[Global], loader: ClassLoader) {
       reachVal(v)
     case Op.Copy(v) =>
       reachVal(v)
-    case Op.Sizeof(ty) =>
+    case Op.Sizeof(ty, retType) =>
```

Comment:
We usually use `ty` as a shortcut for `type` and `retty` name as a shortcut for `return type`.